package-lock.json

```diff
@@ -517,6 +517,15 @@
         "@babel/helper-plugin-utils": "^7.8.0"
```

Comment:
Just a reminder to revisit this and leave only the added dependencies

Reply:
Some dependencies were added because the new `devDependencies` requires it as `dependencies` or as `peerDependencies`. Do you recommend removing it?

Reply:
Done